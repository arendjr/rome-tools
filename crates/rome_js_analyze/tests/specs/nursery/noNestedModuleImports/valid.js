/* should not generate diagnostics */

import "./sibling";
import { someSymbol } from "../aunt";
