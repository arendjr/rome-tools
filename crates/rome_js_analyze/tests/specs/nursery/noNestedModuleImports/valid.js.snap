---
source: crates/rome_js_analyze/tests/spec_tests.rs
assertion_line: 103
expression: valid.js
---
# Input
```js
/* should not generate diagnostics */

import "./sibling";
import { someSymbol } from "../aunt";

```


